### DISCUSSION TITLE
Live visitors and initialize conversation from Agent

### DISCUSSION THREAD

**Discussion by datnguyen293** [chatwoot/chatwoot#2087] [Q&A]
Hi, I want to list live visitors of an inboxes (actually live on the website for example) and then agents can initialize a conversation with a live visitor instead of waiting for visitors to start the conversation.
Can you please give me some hints to accomplish this goals with Chatwoot?

Thanks and best regards,
Dat

**Comment by arealestate** [comment]
I agree, this sounds like a great idea. Maybe it's accomplished with the new 'campaign' work they're mentioning in PRs?

**Reply by pranavrajs** [reply]
Yes, this is correct. Campaigns are being built to accommodate this.

**Reply by datnguyen293** [reply]
Thanks for confirming @pranavrajs 
Do you have plan/expected date for this feature?

Regards,
Dat

**Reply by pranavrajs** [reply]
This should be available in the current release. (May 15 release)

**Reply by rodrigorodriguescosta** [reply]
Hi, is Live visitors feature available? I tried using campaign with no success

**Reply by brundles** [reply]
Hi @pranavrajs - was this released? Ie is there a way via Campaigns to see visitors, so we can proactively engage with them?

**Reply by pgrandmaison** [reply]
Campaigns are working for me, but it looks like it's not possible to setup a campaign to take effect on the entire website. Would it be possible to make https://URL/* or another option to setup a campaign in-effect on all pages of a website?

**Reply by sojan-official** [reply]
@pgrandmaison  we are tracking this on https://github.com/chatwoot/chatwoot/issues/3192

**Reply by pranavrajs** [reply]
@pgrandmaison The wild card is live on 2.12.0. Please see this guide. https://www.chatwoot.com/docs/product/features/campaigns/wild-card-url-in-campaigns

**Comment by brundles** [comment]
Hi @sojan-official 

I really think this is a basic requirement, as you see in most other live-chat platforms. Just having a list of currently online users, and then the ability to click them to start a proactive chat.

I think campaigns with wildcard could be fine - but still just seeing a list of users would also almost be a requirement. Ie. right now the chat agents have **no** idea how many people are in their store, and cannot engage them. 

Campaigns would "only" automate the engagement (which I also love - so please dont kill that idea ;-))

**Reply by pranavrajs** [reply]
> I think campaigns with wildcard could be fine - but still just seeing a list of users would also almost be a requirement. Ie. right now the chat agents have no idea how many people are in their store, and cannot engage them.

Understood. At the moment, it is not possible. With the latest update you can use campaigns in wildcard URL format.

**Reply by adrianmihalko** [reply]
> Hi @sojan-official
> 
> I really think this is a basic requirement, as you see in most other live-chat platforms. Just having a list of currently online users, and then the ability to click them to start a proactive chat.
> 
> I think campaigns with wildcard could be fine - but still just seeing a list of users would also almost be a requirement. Ie. right now the chat agents have **no** idea how many people are in their store, and cannot engage them.
> 
> Campaigns would "only" automate the engagement (which I also love - so please dont kill that idea ;-))

This would be awesome. We need a live tracking for users, example informations:

VISITOR
CURRENT PAGE
CAME FROM
VISITS
CHATS

just like on smartsupp.com

**Comment by GVALFER** [comment]
Yes, I was looking for this feature. 
Really its a basic feature on the livechat today... 
Agents can monitor the users. At the moment I am a little lost, I dont know if the users are online, and if they are, where they are and what page they are seeing.

**Reply by brundles** [reply]
I agree @GVALFER - its a great product, but this is clearly missing

**Comment by kang28ivan** [comment]
I'm also waiting for the Live Visitor feature like on tawkto, 
I'm used to using tawkto and monitoring visitor traffic, when using chatwoot I feel strange because I'm not used to it and have to daydream without seeing the traffic that is happening on the website, in tawkto I can monitor the visitor currently accessing the page.
hopefully this feature will be available soon

**Comment by iamshouvikmitra** [comment]
Any idea if this is something that would be worked on?

**Comment by brundles** [comment]
Still waiting :)

**Comment by brundles** [comment]
Is there any way we can otherwise see where someone is online? Ie. can we see it in the database or some other way? I'd be happy to code whatever is missing.

**Comment by csyedbilal** [comment]
Still waiting :)

**Comment by heidericklucas** [comment]
Still waiting
